README.adoc

```diff
@@ -10,26 +10,26 @@
 :page-layout: guide-multipane
 :page-duration: 30 minutes
 :page-releasedate: 2020-10-30
-:page-description: Explore how to deploy microservices to Red Hat OpenShift 4 by using Kubernetes Operators like the Open Liberty Operator.
+:page-description: Explore how to deploy microservices to Red Hat OpenShift 4 using Kubernetes Operators like the Open Liberty Operator.
 :page-tags: ['Kubernetes', 'Docker', 'Cloud'] 
 :page-permalink: /guides/{projectid}
 :page-related-guides: ['cloud-openshift', 'okd']
 :common-includes: https://raw.githubusercontent.com/OpenLiberty/guides-common/prod
 :imagesdir: /img/guide/{projectid}
 :source-highlighter: prettify
-:page-seo-title: Deploying microservices to Red Hat OpenShift 4 by using Kubernetes Operators
-:page-seo-description: A getting started tutorial with examples on how to deploy cloud-native Java applications and microservices to Red Hat OpenShift cluster by using Kubernetes Operators like the Open Liberty Operator.
+:page-seo-title: Deploying microservices to Red Hat OpenShift 4 using Kubernetes Operators
+:page-seo-description: A getting started tutorial with examples on how to deploy cloud-native Java applications and microservices to Red Hat OpenShift cluster using Kubernetes Operators like the Open Liberty Operator.
```

Comment:
Actually, this one can keep its "by" as it's a longer sentence and reads better.

```suggestion
:page-seo-description: A getting started tutorial with examples on how to deploy cloud-native Java applications and microservices to Red Hat OpenShift cluster by using Kubernetes Operators like the Open Liberty Operator.
```